mobile/src/main/java/butter/droid/fragments/ShowDetailAboutFragment.java

```diff
@@ -61,7 +59,7 @@ public void onCreate(Bundle savedInstanceState) {
     public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
         mRoot = inflater.inflate(R.layout.fragment_detail_about, container, false);
         ButterKnife.bind(this, mRoot);
-        if (VersionUtils.isJellyBean() && container != null) {
+        if (container != null) {
```

Comment:
Yeah, as @blazsolar has said, VersionUtils.isJellyBean was checking with >= so is going to be true at all times (this applies to all changes). 

I can review if the null check is necessary or not